awx/main/tasks.py

```diff
@@ -1172,7 +1172,11 @@ def run(self, pk, **kwargs):
                     del params[v]
 
             if self.instance.is_isolated() is True:
-                playbook = params['playbook']
+                module_args = None
+                if 'module_args' in params:
```

Comment:
@chrismeyersfsu this is one change that was preventing isolated adhoc from working - we've actually got to pass `-m` and `-a`